changelog/eh3jyK_FSMWR2U7RIs-M8w.md

```diff
@@ -0,0 +1,3 @@
+audience: developers
+level: silent
```

Comment:
This probably shouldn't be silent in case people need to modify their configuration if they only read the documentation? Maybe add a note?

Reply:
true! Will add, thanks